code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -355,7 +355,7 @@
 
 /obj/item/gun/ballistic/automatic/surplus
 	name = "Surplus Rifle"
-	desc = "One of countless obsolete ballistic rifles that still sees use as a cheap deterrent. Uses 10mm ammo and its bulky frame prevents one-hand firing."
+	desc = "One of countless obsolete ballistic rifles that still sees use as a cheap deterrent, it seems to be a cheap copy of the Mini-14. Uses 10mm ammo and its bulky frame prevents one-hand firing."
```

Comment:
in-game isn't this supposed to be a mosin nagant?

Reply:
Mosin is its own thing in rifles.dm, the surplus rifle is completely different.

Reply:
There's another surplus rifle that uses 10mm and is semi auto, though using the Mini-14 by name seems meh

Reply:
I might push a sprite change later because holy jesus the current surplus rifle looks like it's straight out of OpenSS13